src/pages/Home.test.js

```diff
@@ -0,0 +1,28 @@
+// Imports React into our test file.
+import React from 'react'
+
+// Imports Enzyme testing and deconstructs Shallow into our test file.
+import Enzyme, { shallow } from 'enzyme'
+
+// Imports Adapter utilizing the latest react version into our test file so we can run a testing render on any component we may need.
+import Adapter from 'enzyme-adapter-react-16'
+
+// Imports in the component we are going to be testing.
+import Home from './Home.js'
+
+//Allows us to utilize the adapter we import in earlier, allowing us to call and render a component.
+Enzyme.configure({ adapter: new Adapter() })
+
+
+describe("When Home renders", () => {
+    let home
+    beforeEach(() => {
+        home = shallow(<Home />)
+    })
+
+    it("displays the homepage content", () => {
+        const renderedHome = home.find("p")
```

Comment:
"p" is a paragraph tag  not the entire render of home please label your variables appropriately.